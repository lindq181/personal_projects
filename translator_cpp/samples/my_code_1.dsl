/*

//Matrix multiply(Matrix A, Matrix B){

//	Matrix C; 

//	if( A.numCols() != B.numRows())
//		print ("incompatible dimensions");
//	else{
//		Int i;
//		Int j;
		//for(i=1:A.numRows()){
		//	for(j=1:B.numCols(){
		//		C[i,j] = 0;
		//	}
		//	Int k;
		//	for(k=1:A.numCols()){
		//		C[i,j] = C[i,j] + A[i, k] * B[k, j];
		//	}
	//	}
	}
//	return C;
//return A;
//}


main(){
//	Matrix A = readMatrix("../samples/myMatrixA.data");
//	Matrix B = readMatrix("../samples/myMatrixB.data");
	Int x = 3;
//	Matrix new_matrix = multiply(A, B);
//
//	print (new_matrix);
} 

*/

main() {
	Int x = 3;
	print(x);
}
