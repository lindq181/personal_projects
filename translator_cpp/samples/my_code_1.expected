4 4
14 26 29 40
20 38 44 53
26 50 59 66
32 62 74 79


